shell/platform/android/BUILD.gn

```diff
@@ -106,7 +106,19 @@ java_library("flutter_shell_java") {
     "io/flutter/embedding/engine/renderer/FlutterRenderer.java",
     "io/flutter/embedding/engine/renderer/OnFirstFrameRenderedListener.java",
     "io/flutter/embedding/engine/dart/DartExecutor.java",
+    "io/flutter/embedding/engine/dart/DartMessenger.java",
     "io/flutter/embedding/engine/dart/PlatformMessageHandler.java",
+    "io/flutter/embedding/engine/systemchannels/AccessibilityChannel.java",
+    "io/flutter/embedding/engine/systemchannels/KeyEventChannel.java",
+    "io/flutter/embedding/engine/systemchannels/LifecycleChannel.java",
+    "io/flutter/embedding/engine/systemchannels/LocalizationChannel.java",
+    "io/flutter/embedding/engine/systemchannels/NavigationChannel.java",
+    "io/flutter/embedding/engine/systemchannels/PlatformChannel.java",
+    "io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java",
+    "io/flutter/embedding/engine/systemchannels/SettingsChannel.java",
+    "io/flutter/embedding/engine/systemchannels/SystemChannel.java",
+    "io/flutter/embedding/engine/systemchannels/SystemChannels.java",
+    "io/flutter/embedding/engine/systemchannels/TextInputChannel.java",
     "io/flutter/embedding/legacy/AccessibilityBridge.java",
     "io/flutter/embedding/legacy/FlutterPluginRegistry.java",
```

Comment:
Is this still used? (I see that it is no longer referenced from FlutterEngine)

Reply:
Removed